README.md

```diff
@@ -2403,6 +2406,52 @@ and 'absent'.
 Name of the ospf router. Valid value is a string.
 
 --
+### Type: cisco_ospf_area
+
+Manages an area for an OSPF router.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I3(1)        | 1.4.0                  |
+| N3k      | 7.0(3)I3(1)        | 1.4.0                  |
+| N5k      | 7.3(0)N1(1)        | 1.4.0                  |
+| N6k      | 7.3(0)N1(1)        | 1.4.0                  |
+| N7k      | 7.3(0)D1(1)        | 1.4.0                  |
+| N8k      | 7.3(0)F1(1)        | 1.4.0                  |
+
+#### Parameters
+
+##### `ensure`
+Determines whether the config should be present or not on the device. Valid values are 'present' and 'absent'.
+
+##### `authentication`
+Enables authentication for the area. Valid values are 'clear_text', 'md5' or 'default'.
+
+##### `default_cost`
+Default_cost for default summary Link-State Advertisement (LSA). Valid values are integer or keyword 'default'.
+
+##### `filter_list_in`
+Filter networks sent to this area. Valid values are string or keyword 'default'.
+
+##### `filter_list_out`
+Filter networks sent from this area. Valid values are string or keyword 'default'.
+
+##### `range`
+Configures an address range for an area.  Valid values are an array of [ip, not_advertise, cost] pairs or keyword 'default'. `ip` is mandatory in the sub-arrays but `not_advertise` and `cost` are optional.

```

Comment:
Range is really about creating summary routes, so how about something like this:

```
Summarizes routes at an area boundary. Optionally sets the area range status to DoNotAdvertise as well as per-summary cost values. Valid values are a nested array of [summary_address, 'not_advertise', cost], or keyword 'default'. The summary-address is mandatory.
```